.github/workflows/code-quality.yml

```diff
@@ -10,7 +10,9 @@ jobs:
       with:
         python-version: "3.12"
     - run: pip install --upgrade pip
-    - run: pip install "ruff<1" "black<23" "isort<6" 
+
+    - run: pip install "ruff" "black" "isort"
```

Comment:
I'm you should lock versions, sometimes it can break workflow due incompatibility in new versions

Use same versions as in poetry